api/src/test/java/org/pmiops/workbench/billing/FreeTierBillingServiceTest.java

```diff
@@ -261,8 +263,7 @@ public void checkFreeTierBillingUsage_twoUsers() {
       DbWorkspaceFreeTierUsage usage = workspaceFreeTierUsageDao.findOneByWorkspace(ws);
       assertThat(usage.getUser()).isEqualTo(ws.getCreator());
       assertThat(usage.getWorkspace()).isEqualTo(ws);
-      assertThat(usage.getCost())
-          .isCloseTo(expectedCosts.get(ws.getWorkspaceId()), withinPercentage(0.001));
+      assertWithinPercentage(usage.getCost(), expectedCosts.get(ws.getWorkspaceId()), 0.001);
```

Comment:
Could this use the `assertCloseEnough` method?

Reply:
`assertCloseEnough()` is written in terms of `assertWithinPercentage`.